firefox-ios/Client/Frontend/Browser/Search/SearchViewController.swift

```diff
@@ -843,6 +817,25 @@ class SearchViewController: SiteTableViewController,
         return cell
     }
 
+    private func configureOneLineCellForSearch(
```

Comment:
Should we maybe rename this method to something like `cellModelForSearch`? From the name it has right now I'd expect the cell to be fully configured instead of it returning a cell model.